content/docs/authorization/_index.md

```diff
@@ -43,7 +43,7 @@ The following diagram shows a high-level overview of CSM for Authorization with
 {{<table "table table-striped table-bordered table-sm">}}
 |               | PowerMax         | PowerFlex | PowerScale |
 |---------------|:----------------:|:-------------------:|:----------------:|
-| Storage Array |PowerMax 2500/8500 PowerMaxOS 10 (6079) , PowerMaxOS 10.0.1 (6079) <br> PowerMax 2000/8000 - 5978.711.xxx, 5978.479.xxx,<br> Unisphere 10.0, 10.0.1|    3.5.x, 3.6.x    | OneFS 8.1, 8.2, 9.0, 9.1, 9.2, 9.3, 9.4, 9.5 |
+| Storage Array |PowerMax 2500/8500 PowerMaxOS 10 (6079) , PowerMaxOS 10.0.1 (6079) <br> PowerMax 2000/8000 - 5978.711.xxx, 5978.479.xxx,<br> Unisphere 10.0, 10.0.1|    3.5.x, 3.6.x, 4.0.x, 4.5    | OneFS 8.1, 8.2, 9.0, 9.1, 9.2, 9.3, 9.4, 9.5 |
```

Comment:
Please remove 3.5.x, 3.6.x for PowerFlex

Reply:
> Please remove 3.5.x, 3.6.x for PowerFlex Replace OneFS 8.1, 8.2, 9.0, 9.1, 9.2, 9.3, 9.4, 9.5 with OneFS 9.3, 9.4, 9.5.0.5, 9.5.0.6

Fixed